app/presenters/api/v2/commodities/commodity_presenter.rb

```diff
@@ -45,7 +45,7 @@ def ancestor_ids
         end
 
         def meursing_code?
-          import_measures.any?(&:meursing?) || export_measures.any?(&:meursing?)
```

Comment:
Meursing measure components only apply to import measure. Reviewed this with @mattlavis-transform